code_challenges/adhoc/counter.py

```diff
@@ -0,0 +1,15 @@
+# create a counter that has an incriment , decremet, and value method (return current value)
+class Counter:
+    def __init__(self):
+        self._value = 0
+        pass
```

Comment:
Pass was here to start because it was need to make an empty function syntactically correct. 

You can't have a ":" at the end of a line without code of some kind in the block below.  It serves no purpose now.